docs/Reference.portuguese.md

```diff
@@ -1636,4 +1637,4 @@ Confira também nosso [Guia de Introdução](Getting-Started.portuguese.md) e [G
 
 Esse arquivo está disponível em:
 
-- [Inglês](Reference.md)
\ No newline at end of file
+- [Inglês](Reference.md)
```

Comment:
I'm confused here... what does it think changed?? This looks identical to me...

Reply:
I did not touch that part of the markdown.
I am not sure why GitHub is marking it as changed.